test/common/modules/live_objects_helper.js

```
@@ -0,0 +1,172 @@
+        // extract object id if present
+        result.objectId = result.objectIds?.[0];
+        return result;
+      }
+
+      throw new Error(
+        `${method}: ${path} FAILED; http code = ${response.statusCode}, error code = ${response.errorCode}, message = ${response.errorMessage}; operation = ${JSON.stringify(opBody)}`,
+      );
+    }
+  }
+
+  return (module.exports = new LiveObjectsHelper());
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)